### PR TITLE
Feature/orca 478 Update s3 policies for deep glacier optional storage

### PR DESCRIPTION
**Pull request by rizbihassan**
## Summary of Changes

Addresses [ORCA-478: Update s3 policies for deep glacier optional storage](https://bugs.earthdata.nasa.gov/browse/ORCA-478)

## Changes

* updated website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md

## Type of change

Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)
- [ x] This change requires a documentation update

## How Has This Been Tested?
deployed the bucket with the new policy in DR account.
Tried writing to the glacier bucket in DR account from the cumulus account.
Write operation succeeded when object storage type were ["GLACIER", "DEEP_ARCHIVE"]
Write operation gave access denied error for other storage types.

## Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x ] Development documentation

## Possible fixes

Since glacier bucket is created in DR account, it cannot be deployed like other modules in ORCA that are deployed in cumulus account. The glacier bucket including its policy has to be deployed together in DR account.

### CODE REVIEW

File: website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md [hunk 1]
```diff
@@ -172,9 +172,10 @@ modifications, which will be detailed below.
       ],
       "Condition": {
         "StringEquals": {
-          "s3:x-amz-acl": "bucket-owner-full-control"
+          "s3:x-amz-acl": "bucket-owner-full-control",
+          "s3:x-amz-storage-class": ["GLACIER", "DEEP_ARCHIVE"]
```
Comment (AndrewDorn) [inline website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md:176]: Remember to add in migration notes when updating the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -577,6 +589,4 @@ None - this is the baseline release.
   * Created some bash scripts for use in the Bamboo build.
   * Updated requirements-dev.txt files for each task and moved the testing framework from nosetest (no longer supported) to coverage and pytest. 
   * Support in GitHub for automated build/test/release via Bamboo
-  * Use `coverage` and `pytest` for coverage/testing
-
-### Changed
```
[the remark this answers is not in the document]
Comment (rizbihassan) [reply CHANGELOG.md:582]: ### Changed was empty that's why removed the header

**Review by AndrewDorn — APPROVED** [review]

File: website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md [hunk 3]
```diff
@@ -172,9 +172,10 @@ modifications, which will be detailed below.
...
         }
-      }
+     }
```
Comment (AndrewDorn) [inline website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md:178]: Looks like the spacing here is a bit off.

**Review by bhazuka — APPROVED** [review]

File: website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md (hunk 3 above)
Comment (rizbihassan) [reply website/docs/developer/deployment-guide/creating-orca-glacier-bucket.md:178]: oops. prettified

**Pull request merged**